packages/config/src/redirects.js

```diff
@@ -11,28 +11,28 @@ const { warnRedirectsParsing } = require('./log/messages')
 // Add `config.redirects`
 const addRedirects = async function (config, logs) {
   const redirectsPath = resolve(config.build.publish, REDIRECTS_FILENAME)
+  const configWithRedirects = await addConfigRedirects(config, redirectsPath, logs)
```

Comment:
This moves the `try`/`catch` block for redirects, the same way we did above for headers.